### PR TITLE
chore(tokens): Improve design tokens import script

### PR DESCRIPTION
**Pull request by pladaria**
Sample execution via github actions:
https://github.com/Telefonica/mistica-web/actions/runs/11702860436

### CODE REVIEW

File: css/vivo-new.css
Comment (yceballost) [inline css/vivo-new.css:1]: Care, these changes should be released "together" with native releases

Comment (yceballost) [inline css/vivo-new.css:1]: you also need to import the new font weight

you can get the font in this jira ticket
https://jira.tid.es/browse/OBVIVO-2687

Download de zip and replace all VivoType fonts (not only medium). All fonts have minor changes in metadata

**Review by yceballost — CHANGES_REQUESTED** [review]

**Review by atabel — APPROVED** [review]

**Review by marcoskolodny — APPROVED** [review]

File: css/vivo-new.css
Comment (yceballost) [reply css/vivo-new.css:1]: I added the zip in this jira

Comment (pladaria) [reply css/vivo-new.css:1]: I've reverted the imported changes

**Review by yceballost — APPROVED** [review]

**Pull request merged**